core/src/avm2/globals/flash/media/AudioOutputChangeReason.as

```diff
@@ -6,6 +6,7 @@
 package flash.media
 {
 
+    // TODO: [API("724")]
```

Comment:
I think mentioning Flash Player 27, and that `724` is just a guess/extrapolation, would be better.